walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +122,72 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
```

Comment:
Another test that's worth considering is one in which the events are explicitly out of order.  Your algorithm relies on the events being sorted (and it does sort them) but eyeballing it I don't see any tests that would fail if that sorting were removed.